src/OWML.ModHelper.Events/HarmonyHelper.cs

```diff
@@ -71,11 +78,13 @@ public void AddPostfix(MethodBase original, Type patchType, string patchMethodNa
 			Patch(original, null, postfix, null);
 		}
 
-		public void EmptyMethod<T>(string methodName) =>
-			EmptyMethod(GetMethod<T>(methodName));
+		public void EmptyMethod<T>(string methodName) => 
+			AddPrefix<T>(methodName, typeof(ClearPatch), "Prefix");
+			//EmptyMethod(GetMethod<T>(methodName));
```

Comment:
don't need these comments?

Reply:
I kept EmptyMethod Transpile as comment in case it will start working again